src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -98,6 +98,18 @@ lemma euclidean_space.nnnorm_eq {𝕜 : Type*} [is_R_or_C 𝕜] {n : Type*} [fin
   (x : euclidean_space 𝕜 n) : ∥x∥₊ = nnreal.sqrt (∑ i, ∥x i∥₊ ^ 2) :=
 pi_Lp.nnnorm_eq_of_L2 x
 
+lemma euclidean_space.dist_eq {𝕜 : Type*} [is_R_or_C 𝕜] {n : Type*} [fintype n]
+  (x y : euclidean_space 𝕜 n) : dist x y = (∑ i, dist (x i) (y i) ^ 2).sqrt :=
+by convert pi_Lp.dist_eq_of_L2 x y
+
+lemma euclidean_space.nndist_eq {𝕜 : Type*} [is_R_or_C 𝕜] {n : Type*} [fintype n]
+  (x y : euclidean_space 𝕜 n) : nndist x y = (∑ i, nndist (x i) (y i) ^ 2).sqrt :=
+by convert pi_Lp.nndist_eq_of_L2 x y
+
+lemma euclidean_space.edist_eq {𝕜 : Type*} [is_R_or_C 𝕜] {n : Type*} [fintype n]
```

Comment:
Shouldn't this be a `edist_eq_of_L2` lemma as well?

Reply:
I thought not because there is no `ennreal.sqrt`, but actually you still can simplify ` ^ (2 : ℝ)` to `^ (2 : ℕ)`.